src/components/header/Header.tsx

```diff
@@ -0,0 +1,58 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import logo from '../../../public/logo.svg';
+import smallBannerImage from '../../../public/banners/Breakpoint=Small.png';
+import mediumBannerImage from '../../../public/banners/Breakpoint=Medium.png';
+import largeBannerImage from '../../../public/banners/Breakpoint=Large.png';
+
+export const bannerImages = [
+  {
+    src: smallBannerImage,
+    alt: 'Banner for small screens',
+    classes: 'block sm:hidden',
+    width: 375,
+    height: 300,
+  },
+  {
+    src: mediumBannerImage,
+    alt: 'Banner for medium screens',
+    classes: 'hidden sm:block md:hidden',
+    width: 768,
+    height: 350,
+  },
+  {
+    src: largeBannerImage,
+    alt: 'Banner for large screens',
+    classes: 'hidden md:block',
+    width: 1440,
+    height: 400,
+  },
+];
+
+const Header = () => {
+  return (
+    <header>
+      <nav className='sticky top-0 z-10 mx-auto flex bg-white p-4 lg:ps-16'>
```

Comment:
🟡 I think a better practice would be to split this component into two: one for the navigation bar (with the potential navigation links in the future) and the banner image

Reply:
I believe keeping it as a single unit is more appropriate in this case. We're not planning on reusing these elements, and keeping them together simplifies testing and maintains a clear connection between the navigation and the banner image.